src/extensions/bigbang/manifests.go

```diff
@@ -63,12 +63,23 @@ kyvernoPolicies:
           - zarf # don't have Kyverno prevent Zarf from doing zarf things
           `
 
-func manifestZarfCredentials(version string) corev1.Secret {
+const bbV2ZarfYoloModeCredentialsValues = `
+registryCredentials:
+  registry: "registry1.dso.mil"
+  username: "###ZARF_VAR_REGISTRY1_USERNAME###"
+  password: "###ZARF_VAR_REGISTRY1_CLI_SECRET###"
```

Comment:
While doing this works, it requires the user to create `variables` at the package level which might not be intuitive.  This may be better done via an override of the `manifests` key for flux's secrets and an extra values.yaml for the YOLO mode.  That would give more flexibility too for different ways to bring in these secrets than these explicit ###ZARF_VARs

Reply:
To explain that more when you specify the `bigbang` noun you can also specify `manifests` on the same component as well and they will be merged together which should work for flux.

Reply:
I see where you're coming from - having variables deep within the Zarf source code feels a little bit weird to me as well. Trying your suggestion (see below) yielded similar results, where the initial deployment hangs as it is unable to send the flux `private-registry` secret beforehand. 

```yaml
...
components:
  - name: bigbang
    required: true
    manifests:
      - name: private-registry
        namespace: flux-system
        files:
          - private-registry.yaml
    extensions:
      bigbang:
        version: 2.0.0
        valuesFiles:
          - ingress.yaml
          - kyverno.yaml
          - loki.yaml
```

Reply:
Something like the following seems to work:

```yaml
components:
  - name: flux
    manifests:
      - name: flux-system
        namespace: flux-system
        files:
          - private-registry.yaml
  - name: bigbang
    required: true
    extensions:
      bigbang:
        version: 2.0.0
        valuesFiles:
          - ingress.yaml
          - kyverno.yaml
          - loki.yaml
```

Reply:
Removed internal use of Zarf Variables in latest commits